apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -121,22 +81,11 @@ export const themeOptions = mergeTheme({
       alignItems: 'center',
       marginBottom: '20px',
     },
-    headline3: {
-      fontWeight: '700',
-      fontSize: '18px',
-      lineHeight: '20px',
-    },
-    body1: {
-      fontWeight: 400,
+    subtitle1: {
+      fontWeight: 500,
```

Comment:
Should the `fontWeight` be 400 for `subtitle1`, if it replaces `body1`?

Reply:
The Figma designs use 500 wherever this style is used. I think it was incorrect in the app in some cases.